cypress/integration/main_spec.coffee

```diff
@@ -141,14 +141,22 @@ describe "Main", ->
       cy.get("aside#sidebar")
         .should("be.visible")
 
-    it "has table of contents and goes to 0.19.0", ->
-      cy.get("aside#article-toc")
-        .should("be.visible")
-        .wait(2000) # allows menuspy to load and set the menu links
-        .contains("0.19.0")
-        .click()
-      cy.url()
-        .should('include', '#0-19-0')
+    it "has a populated table of contents", ->
+      if Cypress.config('baseUrl').includes('localhost')
```

Comment:
I'm not sure if we do this on the docs... but in other projects we set a environment variable in Cypress to be that of `process.env.NODE_ENV` so that you can go off of that.

For instance by using `cypress`plugins/index.js` you can use the configuration API...

```js
module.exports = (on, config) => {
  // set the env to be that of NODE_ENV
  config.env.NODE_ENV = process.env.NODE_ENV || 'development'
}
```

Now in your tests you can do...

```js
if Cypress.env('NODE_ENV') ...
```

Reply:
Wasn't able to get this wired up correctly, but seems like it should've been an easy fix. Think I'd vote for merging as is and I can dig into adding this to the docs when I'm blocked on other stuff.

Reply:
No worries - I think i'll hack on it so it doesn't get lost in the shuffle and you'll see what I mean.